reflect_test.go

```diff
@@ -101,6 +101,9 @@ type TestUser struct {
 	Rank       int     `json:"rank,omitempty" jsonschema:"enum=1,enum=2,enum=3"`
 	Multiplier float64 `json:"mult,omitempty" jsonschema:"enum=1.0,enum=1.5,enum=2.0"`
 
+	// Tests for simple enum tags with space
+	Family      string  `json:"family,omitempty" jsonschema:"enum=kid, enum=mother, enum=father"`
```

Comment:
Rather than updating the primary structures, I'd suggest adding an independent test at the end of this file for the specific use-case.